flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/KubernetesOperatorConfigOptions.java

```diff
@@ -369,6 +395,13 @@ public static String operatorConfigKey(String key) {
                     .withDescription(
                             "Type of the binary format in which a savepoint should be taken.");
 
+    @Documentation.Section(SECTION_DYNAMIC)
+    public static final ConfigOption<CheckpointType> OPERATOR_CHECKPOINT_TYPE =
+            operatorConfig("checkpoint.type")
+                    .enumType(CheckpointType.class)
+                    .defaultValue(CheckpointType.FULL)
+                    .withDescription("Type of checkpoint.");
```

Comment:
Let's list the options here, i.e. full and incremental.

Reply:
Added automatically-generated description to the docs.